custom/icds_reports/utils/__init__.py

```diff
@@ -16,10 +16,11 @@
 from base64 import b64encode
 from io import BytesIO
 from dateutil.relativedelta import relativedelta
+from django.conf import settings
 from django.template.loader import render_to_string, get_template
 from openpyxl.styles import PatternFill, Border, Side, Alignment, Font
 from openpyxl import Workbook
-from xhtml2pdf import pisa
```

Comment:
does this mean `xhtml2pdf` is no longer used? Can you remove that from requirements.in and add it to uninstall-requirements.txt

Reply:
Yes, it was the last place where we used it. I have removed it from the requirements.

Reply:
I think you still need to add it to `uninstall-requirements.txt` as emord said

Reply:
I have updated that and fixed conflicts.